src/parser/estree_translator.ml

```diff
@@ -966,6 +966,7 @@ end with type t = Impl.t) = struct
       | Literal.Boolean b -> bool b
       | Literal.Null -> null
       | Literal.Number f -> number f
+      | Literal.BigInt _ -> null
```

Comment:
So this isn't quite what I mean. I think you need to add a new node for bigint literals. You can either try and follow the estree proposal or you can copy what babylon does.

[Babylon outputs a node like](https://github.com/babel/babel/blob/master/packages/babel-parser/test/fixtures/experimental/bigint/valid-hex/output.json)

```
        "expression": {
          "type": "BigIntLiteral",
          "start": 0,
          "end": 9,
          "loc": {
            "start": {
              "line": 1,
              "column": 0
            },
            "end": {
              "line": 1,
              "column": 9
            }
          },
          "extra": {
            "rawValue": "0xFFF123",
            "raw": "0xFFF123n"
          },
          "value": "0xFFF123"
        }
```

Whereas estree would be something like

```
"expression": {
  "type": "BigIntLiteral",
  "start": 0,
  "end": 9,
  "loc": {
  "start": {
    "line": 1,
    "column": 0
  },
  "end": {
    "line": 1,
    "column": 9
  },
  "value": 0xFFF123n;
  "bigint": "0xFFF123n";
}, 
```

Reply:
Then how should be named node for type literals?